keps/sig-cluster-lifecycle/generic/draft-20200508-communicating-a-local-registry/kep.yaml

```diff
@@ -0,0 +1,11 @@
+title: Standard for communicating a local registry
+kep-number: NNNN
+authors:
+  - "@nicks"
+owning-sig: sig-cluster-lifecycle
+status: provisional
+creation-date: 2020-05-08
+reviewers:
+  - TBD
```

Comment:
you can add me here for now, but ideally once more people comment you can add them too.

Reply:
done